apis/generate.go

```diff
@@ -20,13 +20,13 @@ limitations under the License.
 // https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
 
 // Remove existing CRDs
-//go:generate rm -rf ../cluster/charts/crossplane/templates/crds
+//go:generate rm -rf ../cluster/charts/crossplane-types/crds
```

Comment:
should this clean up step not delete from both locations?

Reply:
`crossplane-types/crds` is the main place where kubebuilder generates the CRD. So it deletes that directory before dumping CRDs. 
Copying to other locations happen as part of `make` so it make sense to delete them there. Updated makefile to cleanup those directories before copying which also resolves your concern about zombie files in comment: https://github.com/crossplaneio/crossplane/pull/1139/files#r372736521